### DISCUSSION TITLE
Welcome to homeassistant_electrolux_status Discussions!

### DISCUSSION THREAD

**Discussion by albaintor** [albaintor/homeassistant_electrolux_status#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by wgumaa** [comment]
Thanks @albaintor for setting this up! Let's get this up and working!

**Comment by slingshot74** [comment]
@albaintor Thank you very much for you work!

**Comment by slingshot74** [comment]
I get an update wit h a bit of strange from and to version.
The update is from 6cff18e -> 54adc01
Is this correct?

Unfortunately I get a 403 forbidden error

**Comment by albaintor** [comment]
No this is an intermediate commit, not working. I have added a new logic if capabilities cannot be extracted => I use the internal catalog against reported data. This is working but I need to fix bugs

**Comment by juhanjuku** [comment]
Just found your work and can confirm - it works with my AEG owen BPE748380M
![image](https://github.com/albaintor/homeassistant_electrolux_status/assets/12792066/d7588160-24cc-430e-a0ea-7c5a426f3748)

**Comment by RogValk** [comment]
@albaintor Is your fork life and working, looking at the above comment from last week?
I am at the point of buying a new washing machine and want to be able to control it in the future from Home Assistant because of solar energy availability. Dubbing on AEG or Samsung, where I also have seen an video for using Samsung Home Connect Alt app from HACS.

**Reply by albaintor** [reply]
Hi, which comment ?
I can't assure you which appliance will work or be compatible with this component. Normally they should use the same platform/framework

**Comment by alimdi** [comment]
Hi,
How to add configuration information as the config flow don't seem to fire up ?
Thank you

**Comment by albaintor** [comment]
> Hi, How to add configuration information as the config flow don't seem to fire up ? Thank you

Hi, not sure to understand your problem : after installing the integration, you should be able to click on the integration and add a device. Please share screenshots

**Reply by alimdi** [reply]
Hi,
The probleme was indeed on my side as the ad blocker was bloking the the config flow popup.
Once deactivated, everything is working like a charme.
Thank you for the work making this addon available

**Comment by LoznOz** [comment]
Hi firstly big thank-you for your work in creating this.
I can confirm it works with our Electrolux Washing Machine **EWF9042R7WB** Ultimate Care 700.

![Electrolux Integration Device Info screenshot](https://github.com/albaintor/homeassistant_electrolux_status/assets/119017698/5309c8ed-5ade-4a88-a1c7-0b896b0b6e4f)
![Electrolux Integration Sensors screenshot](https://github.com/albaintor/homeassistant_electrolux_status/assets/119017698/4df9b153-3de3-4566-8770-743a858c893d)
![Electrolux Integration Diagnostic screenshot](https://github.com/albaintor/homeassistant_electrolux_status/assets/119017698/361fe1d5-9c07-4d6b-ae27-50bf8ef2aec9)
![Electrolux Integration Controls 1of2 screenshot](https://github.com/albaintor/homeassistant_electrolux_status/assets/119017698/0a44b1ae-529c-4eda-a561-7ee9f204d173)
![Electrolux Integration Controls 1of2 screenshot](https://github.com/albaintor/homeassistant_electrolux_status/assets/119017698/d3ecbd44-16d4-4d55-a99c-89cdc757284d)

**Comment by Ztaeyn** [comment]
Hello.

Confirmed to work on AEG BFB8000QM (combi micro oven) and BFS8800M (steam oven)

**Comment by klapecm** [comment]
Hello,
thank you for the integration. I have noob questions. What username and country code to use?

username does not seem to be email address registered for electrolux devices.
country_code for Europe (Czech)  is emea or cs?

I have tried multiple combinations of Electrolux credentials.

<img width="910" height="658" alt="image" src="https://github.com/user-attachments/assets/289ce898-c633-4f8e-a6ed-2f3686c7f71e" />

Thank you

**Comment by necromitt** [comment]
Hello, from some time i cant see anything. For all entities is shown warning message: "_This entity is no longer being provided by the electrolux_status integration. If the entity is no longer in use, delete it in settings._" Is there some problem with integration? And same problem as @klapecm with country code for Czech republic. CS, CZ did not work

Thanks a lot
